readme.adoc

```diff
@@ -622,6 +622,62 @@ https://github.com/aws-samples/aws-microservices-deploy-options/issues/66
 
 . Open the https://us-west-2.console.aws.amazon.com/xray/home?region=us-west-2#/service-map[X-Ray console] and watch the service map and traces.
 
+=== Monitoring: Conduit
+https://conduit.io/[Conduit] is a small, ultralight, incredibly fast service mesh centered around a zero config approach. It can be used for gaining remarkable visibility in your Kubernetes deployments.
+
+. Confirm that both Kubernetes client and server versions are v1.8.0 or greater using `kubectl version --short`
+
+. Install the Conduit CLI on your local machine:
+
+  curl https://run.conduit.io/install | sh
+
+. Add the `conduit` command into your PATH:
+
+  export PATH=$PATH:$HOME/.conduit/bin
+
+. Verify the CLI is installed and running correctly. You will see a message that says 'Server version: unavailable' because you have not installed Conduit in your deployments.
+
+  conduit version
+
+. Install Conduit on your Kubernetes cluster. It will install into a separate `conduit` namespace, where it can be easily removed.
+
+  conduit install | kubectl apply -f -
+
+. Verify installation of Conduit into your cluster. Your Client and Server versions should now be the same.
+
+  conduit version
+
+. Verify the Conduit dashboard opens and that you can connect to Conduit in your cluster.
+
+  conduit dashboard
+
+. Install the demo app to see how Conduit handles monitoring of your Kubernetes applications.
+
+  curl https://raw.githubusercontent.com/runconduit/conduit-examples/master/emojivoto/emojivoto.yml | conduit inject - | kubectl apply -f -
```

Comment:
Can this be adapted to the application used in this repo? See the example at https://github.com/aws-samples/aws-microservices-deploy-options#deployment-canary-deployment-with-istio

Reply:
Not without significant work. There is a traffic creating service that drives real traffic through the demo namespace so that a participant can interact with and troubleshoot real service requests happening in real time. That may be possible in the future, but for now the way to get a comprehensive troubleshooting experience is to use the demo app.

Reply:
That's the benefit, yes. I was writing this per your request in #246 for a telemetry example using Conduit. Without a fair amount of live traffic flowing through the application (i.e. more than what one person clicking around would show), Conduit telemetry will just show zero (or near zero) real time statistics.

Something might be possible with the sample app, but that would require setting up another service whose sole job is to generate constant traffic to the sample app.

Reply:
I should've qualified that the microservices to be used are relevant to this repo :)

Anyway, will merge this request for now. But will open another issue to update sample to use microservices from this repo, ok?

Reply:
Thanks, @arun-gupta. That'll work.